app.rb

```diff
@@ -0,0 +1,198 @@
+students = [
+    {
+        first_name: 'Ahmed',
+        last_name: 'Al-Bahrani'
+    },
+    {
+        first_name: 'Reem',
+        last_name: 'AlHarbi',
+    },
+    {
+        first_name: 'Mansour',
+        last_name: 'Almohsen',
+    }
+  ]
+  
+  upper_case_full_names = []
+
+  students.length.times do |i|
+    upper_case_full_names.push( students[i][:first_name].upcase + " "+ students[i][:last_name].upcase) ) 
```

Comment:
there is an extra parenthesis at the end of this line that's causing you an error